src/evaluate/hub.py

```diff
@@ -0,0 +1,63 @@
+from huggingface_hub.repocard import metadata_update
+
+
+def push_to_hub(
+    to: str,
+    task_type: str,
+    metric_value: float,
+    metric_type: str,
+    dataset_name: str,
+    dataset_type: str,
+    task_name: str = None,
+    dataset_config: str = None,
+    dataset_split: str = None,
+    dataset_revision: str = None,
+    dataset_args: dict[str, int] = None,
+    metric_name: str = None,
+    metric_config: str = None,
+    metric_args: dict[str, int] = None,
+):
+    """
+    TODO: Add documentation
```

Comment:
I think it is crucial to document these well with examples, as it is not very clear what the difference between name and type is. Name being the pretty name and type being the name to load them.